primitives/src/channel_v5.rs

```diff
@@ -0,0 +1,15 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{BigNum, ChannelId, ValidatorId, Address};
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Channel {
+    pub id: ChannelId,
+    pub leader: ValidatorId,
+    pub follower: ValidatorId,
```

Comment:
Currently `validatorId === Address`, so why not make `ValidatorId` or `Address`  a type alias?

Reply:
We do not want to be able to substitute a `ValidatorId` with an `Address`, `ValidatorId` != `Address`.
If you need to access the bytes or even convert the ValidatorId to the inner Address, that's fine, but it must be explicit.